modules/nw-secondary-ext-gw-configure.adoc

```diff
@@ -0,0 +1,138 @@
+----
+$ oc create -f <file>.yaml
+----
++
+--
+where:
+
+`<file>`:: Specifies the name of the YAML file that you created in the previous step.
+--
++
+.Example output
+[source,text]
+----
+adminpolicybasedexternalroute.k8s.ovn.org/default-route-policy created
+----
+
+. To confirm that the admin policy based external route was created, enter the following command:
++
+[source,terminal]
+----
+$ oc describe apbexternalroute <name> | tail -n 6
+----
++
+--
+where:
+
+`<name>`:: Specifies the name of the `AdminPolicyBasedExternalRoute` object.
+--
++
+.Example output
+[source,text]
+----
+Status:
+  Last Transition Time:  2023-04-24T15:09:01Z
+  Messages:
+  Configured external gateway IPs: 172.18.0.8
+  Status:  Success
+Events:  <none>
+----
+
+.Verification
+
+If you created an `AdminPolicyBasedExternalRoute` object that selects a host-network pod IP address as the secondary external gateway, you can confirm that the next hop is correct for a pod with the following steps:
+
+. To get the IP address of the pod, enter the following command:
++
+[source,terminal]
+----
+oc get pods/<pod_name> -n <namespace> -o wide
+----
++
+--
+where:
+
+`<pod_name>`:: Specifies the name of the pod.
+`<namespace>`:: Specifies the namespace of the pod.
+--
++
+.Example output
+[source,text]
+----
+NAMESPACE  NAME   READY   STATUS      RESTARTS      AGE   IP            NODE      NOMINATED NODE   READINESS GATES
+ns1        pod1   1/1     Running     1 (37m ago)   41m   10.130.0.8    node1     <none>           <none>
+----
+
+. Confirm that the IP address from the previous step is available as an external gateway.
+
+.. To find the the OVN-Kubernetes control plane pod that manages the next hop for the pod, enter the following command:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-ovn-kubernetes \
+  --field-selector spec.nodeName=<node_name> \
+  -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | \
+    grep ovnkube-node-
+----
++
+--
+where:
+
+`<node_name>`:: Specifies the name of the node from the `NODE` column that the pod from the previous step is running on.
+--
++
+.Example output
+[source,text]
+----
+ovnkube-node-rpt55
+----
+
+.. To confirm that the OVN-Kubernetes node pod includes the correct next hop, enter the following command:
++
+[source,terminal]
+----
+$ oc exec -t <pod_name> -n openshift-ovn-kubernetes  -c nbdb \
+    -- ovn-nbctl lr-route-list GR_ovn-work | grep <pod_ip> -A 6 -B 4
+----
++
+--
+where:
+
+`<pod_name>`:: Specifies the name of the OVN-Kubernetes node pod from the previous step.
+`<pod_ip>`:: Specifies the name of the pod IP address.
+--
++
+.Example output
+[source,text]
+----
+_uuid             : e92484e8-f945-4ffa-be1f-de6c724dca20
+bfd               : []
+external_ids      : {}
+ip_prefix         : "10.128.3.229/32"
+nexthop           : "172.18.0.8"
+options           : {ecmp_symmetric_reply="true"}
+output_port       : rtoe-GR_ovn-control-plane
+policy            : src-ip
+route_table       : ""
+----
```

Comment:
@jab-rh The data might be correct but format isn't, what I mean to so this might be valid output but not from current command on L#115, unless maybe you used some formatting option(?) missing on L#115. 

Below is what I got, adapted to suit my env
$oc exec -ti $POD -n openshift-ovn-kubernetes -c nbdb -- ovn-nbctl lr-route-list GR_worker-0-1 | grep 10.128.2.206 -A6 -B4

IPv4 Routes
Route Table <main>:
             10.128.2.206               172.18.0.10 src-ip rtoe-GR_worker-0-1 ecmp-symmetric-reply bfd
             10.128.3.229               172.18.0.10 src-ip rtoe-GR_worker-0-1 ecmp-symmetric-reply bfd
         169.254.169.0/29             169.254.169.4 dst-ip rtoe-GR_worker-0-1
            10.128.0.0/14                100.64.0.1 dst-ip
                0.0.0.0/0             192.168.123.1 dst-ip rtoe-GR_worker-0-1

Reply:
@tshefi, I pulled it from the original Google slide deck for this feature; I guess I copied the wrong output or the deck itself is wrong.

Reply:
I don't know if that route table listing helps confirm that this feature is working correctly?

I guess a better question is: Is there a verification procedure that helps a user see that they have succeeded at this? What I've used came from the slide deck, but I guess I copied it wrong or it isn't the proper way? What can I use instead that is correct?

@jordigilh, is it possible for someone to introspect to determine this feature is working correctly? If so, how to achieve this? Thanks!

Thanks!